smithy-kotlin-codegen/build.gradle.kts

```diff
@@ -27,6 +27,7 @@ dependencies {
     api("software.amazon.smithy:smithy-waiters:$smithyVersion")
     implementation("software.amazon.smithy:smithy-aws-traits:$smithyVersion")
     implementation("software.amazon.smithy:smithy-protocol-test-traits:$smithyVersion")
+    implementation("org.jsoup:jsoup:1.14.1")
```

Comment:
add `jsoupVersion` to `gradle.properites` with the other `codegen` related versions and reference it the way we do other versions here